templates/cinder/bin/init.sh

```diff
@@ -41,18 +41,26 @@ cp ${DEFAULT_DIR}/* ${MERGED_DIR}
 cp -a ${SVC_CFG} ${SVC_CFG_MERGED}
 
 cp ${DEFAULT_DIR}/cinder.conf ${SVC_CFG_MERGED_DIR}/00-default.conf
+
```

Comment:
Maybe add the empty lines in the previous commit.

Reply:
Ack.